test_scripts/Security/GetSystemTime/common.lua

```diff
@@ -0,0 +1,164 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local security = require("user_modules/sequences/security")
+local utils = require("user_modules/utils")
+local test = require("user_modules/dummy_connecttest")
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+--[[ General configuration parameters ]]
+config.serverCertificatePath = "./files/Security/GetSystemTime_certificates/spt_credential.pem"
+config.serverPrivateKeyPath = "./files/Security/GetSystemTime_certificates/spt_credential.pem"
+config.serverCAChainCertPath = "./files/Security/GetSystemTime_certificates/spt_credential.pem"
+
+--[[ Module ]]
+local m = actions
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 3
+config.isCheckClientCertificate = false
+config.application1.registerAppInterfaceParams.appName = "server"
+config.application1.registerAppInterfaceParams.appID = "SPT"
+m.appHMIType = "DEFAULT"
+config.application1.registerAppInterfaceParams.appHMIType = { m.appHMIType }
+
+--[[ Variables ]]
+m.frameInfo = security.frameInfo
+m.delayedExp = utils.wait
+m.readFile = utils.readFile
+
+--[[ Functions ]]
+local function getSystemTimeValue()
```

Comment:
Seems that relying on the current time, these tests will inevitably fail in the future because several of the certificates will have expired, a dummy value should likely just be returned instead.

Reply:
@jacobkeeler Please check 649be65